### PR TITLE
Fix profile switching

### PR DESCRIPTION
**Pull request by haslinghuis**
Fixes: #2997

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

File: src/js/tabs/pid_tuning.js
Comment (Asizon) [inline src/js/tabs/pid_tuning.js:2496]: Why we need a timer here now @haslinghuis ?

**Review by Asizon — CHANGES_REQUESTED** [review]

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [reply src/js/tabs/pid_tuning.js:2496]: Assumed it was missing a timer in the interval code?

Comment (haslinghuis) [reply src/js/tabs/pid_tuning.js:2496]: It was missing a timer (bug) in the interval code. Without type checking like TS, JS did allow it to run.

Function is defined as 
```
GuiControl.prototype.interval_add = function (name, code, interval, first) {
```

**Review by Asizon — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Pull request merged**
